tools/StaticAnalysis/Exceptions/AzureRM.Network/SignatureIssues.csv

```diff
@@ -270,3 +270,11 @@
 "c:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Network\Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.NewAzureRmVirtualHubRouteTableCommand","New-AzureRmVirtualHubRouteTable","1","8100","New-AzureRmVirtualHubRouteTable Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "D:\0PS\EA\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Network\Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.NewAzureApplicationGatewayFirewallExclusionConfigCommand","New-AzureRmApplicationGatewayFirewallExclusionConfig","1","8100","New-AzureRmApplicationGatewayFirewallExclusionConfig Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"c:\wkpc\PsPri-OnDemand\src\Package\Release\ResourceManager\AzureResourceManager\AzureRM.Network\Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.NewAzureRmExpressRoutePort","New-AzureRmExpressRoutePort","1","8410","Parameter BandwidthInGbps of cmdlet New-AzureRmExpressRoutePort does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"c:\wkpc\PsPri-OnDemand\src\Package\Release\ResourceManager\AzureResourceManager\AzureRM.Network\Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.RemoveAzureRmExpressRoutePort","Remove-AzureRmExpressRoutePort","1","8600","Cmdlet 'Remove-AzureRmExpressRoutePort' has no defined output type.","Add an OutputType attribute that declares the type of the object(s) returned by this cmdlet. If this cmdlet returns no output, please set the output type to 'bool' and make sure to implement the 'PassThru' parameter."
+"c:\wkpc\PsPri-OnDemand\src\Package\Release\ResourceManager\AzureResourceManager\AzureRM.Network\Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.NewAzureExpressRouteCircuitCommand","New-AzureRmExpressRouteCircuit","1","8410","Parameter BandwidthInGbps of cmdlet New-AzureRmExpressRouteCircuit does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"c:\wkpc\PsPri-OnDemand\src\Package\Release\ResourceManager\AzureResourceManager\AzureRM.Network\Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.NewAzureExpressRouteCircuitCommand","New-AzureRmExpressRouteCircuit","1","8510","Cmdlet 'New-AzureRmExpressRouteCircuit' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
```

Comment:
It would be better to add default parameter set code to avoid cmdlet usage issues than add these entries to exclusions